porch/docs/porch-roadmap.md

```diff
@@ -0,0 +1,82 @@
+# Porch Roadmap
+
+This document outlines next steps for Porch in several areas. This is a living
+document and as of its initial creation is not yet complete.
+
+## API Changes
+
+* Expression kpt function type 'signature', including:
+  * types of resources the function acts on
+  * schema of the function config
+  * types of resources the function _produces_ (if the function introduces new
+    resources into the configuration package)
+* Add `Package` resource to encapsulate all `PackageRevision`s of the same
+  package, link to the latest `PackageRevision`. Possibly the `Package` resource
+  can allow mutation of the package or its contents, and automatically create
+  new (_Draft_) `PackageRevision` resources on mutations.
+* Implement support for API-level filtering (field and label selectors) for all
+  `list` operations.
+
+## Repository Management
+
+* Complete the OCI support (current implementation is partial, misisng package
```

Comment:
typo: misisng -> missing

Reply:
Thanks! fixed. I also pasted all the text into Google Doc to check for spelling and fixed all issues that came up. Hopefully this will help